src/smallstack.rs

```diff
@@ -207,26 +207,28 @@ impl<'a> VM<'a> {
 }
 
 fn main() {
+    let mut content = String::new();
```

Comment:
What's your convention? Indentation in the whole file is a bit dodgy. Two white spaces? Four? Tabs?